docs/tutorials/Monge_Gap.ipynb

```diff
@@ -1,5 +1,15 @@
 {
```

Comment:
<p>Links, such as <code>ott.solvers.nn.losses.monge_gap</code>,MapEstimator and W2NeuralDual need to be adjusted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/467/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>